src/discord_bot/bot.py

```diff
@@ -42,18 +35,21 @@ async def my_long_command_function(ctx: interactions.SlashContext) -> None:
   name="prompt", description="Enter your prompt", required=True, opt_type=interactions.OptionType.STRING, min_length=10
 )
 async def ask_model(ctx: interactions.SlashContext, model: str = "", prompt: str = "") -> None:
-  await ctx.send(f'You asked model {model} with the prompt: "{prompt}"')
+  if model not in MODEL_CHOICES:
```

Comment:
Is this covered by https://github.com/bifrostlab/llm-assistant/pull/32 already. Don't pull from different PR in before merging.

Reply:
That's why I put it in Draft mode sir